Gemfile.lock

```diff
@@ -214,6 +214,7 @@ GEM
 
 PLATFORMS
   ruby
+  x86_64-darwin-17
```

Comment:
why is this necessary?

Reply:
This is just a newer bundler config thing. It isn't "necessary" but its something that gets automatically added with newer bundler versions.